onnxruntime/core/flatbuffers/schema/ort.fbs

```diff
@@ -121,11 +126,12 @@ table ValueInfo {
   type:TypeInfo;
 }
 
-// TODO add support of SparseTensor/Opaque
+// TODO add support of Opaque
 union TypeInfoValue {
   tensor_type:TensorTypeAndShape,
   sequence_type:SequenceType,
   map_type:MapType,
+  sparse_tensor_type:SparseTensorType
 }
```

Comment:
Do we need to support sparse tensor here if it can only be an initializer (or attribute of a Constant node)?